From 627adc167c97c6fca4401e7d05ace1e288583e04 Mon Sep 17 00:00:00 2001
From: Chris Mayo <aklhfex@gmail.com>
Date: Tue, 7 Jun 2016 20:01:55 +0100
Subject: [PATCH 1/7] fix application icon does not show on GNOME desktop

- phoenix.pro renames fritzing_icon.png to fritzing.png
- Icon value should be either an absolute path or icon name

(cherry picked from commit f712d0139e3a0b1146566c989a3d523da9cdcd42)
---
 fritzing.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fritzing.desktop b/fritzing.desktop
index 854319e..1791c92 100644
--- a/fritzing.desktop
+++ b/fritzing.desktop
@@ -4,7 +4,7 @@ Name=Fritzing
 GenericName=Fritzing
 Comment=Electronic Design Automation software
 Exec=Fritzing
-Icon=icons/fritzing_icon.png
+Icon=fritzing
 Terminal=false
 Type=Application
 Categories=Development;IDE;Electronics;EDA;
-- 
2.8.3

